.github/workflows/helm-lint.yaml

```diff
@@ -0,0 +1,13 @@
+name: Helm lint
+on:
+  push:
+
+jobs:
+  helm-lint:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Lint Helm chart
+        uses: bakdata/ci-templates/actions/helm-lint@v1.10.0
+        with:
+          lint-config-path: ".github/lint-config.yaml"
```

Comment:
This file does not exist

Reply:
https://github.com/bakdata/kafka-connect-resetter/blob/main/.github/lint-config.yaml

Reply:
Ah I expected it to be added here

Reply:
It was added for the release workflow because it also lints before releasing